eclair-core/src/main/resources/reference.conf

```
@@ -25,8 +25,8 @@ eclair {
     rpcpassword = "bar"
     // Name of the bitcoind wallet that should be used to fund channels.
     // Once set you should NOT change it if your node has channels open, otherwise you may lose funds.
-    // NB: leave empty to automatically select the default loaded wallet.
-    wallet = ""
+    // NB: leave undefined to automatically select the default loaded wallet.
+    #wallet = ""
```

I'm not sure what `#` does (it's not obvious how to find the documentation for that syntax) and why we need it...can you expand on that?

---

It's just a comment, meaning that the variable is undefined by default. Not sure why I didn't use `//`

```suggestion
    //wallet = ""
```